astropy/units/physical.py

```diff
@@ -527,4 +528,32 @@ def get_physical_type(obj):
 for unit, physical_type in _units_and_physical_types:
     def_physical_type(unit, physical_type)
 
+
+# This generates a docstring addition for this module that describes all of the
+# standard physical types defined here.
+if __doc__ is not None:
+    docstring = io.StringIO()
+
+    docstring.write("""
+
+.. list-table:: Defined Physical Types
+   :header-rows: 1
+   :widths: 30 10 50
```

Comment:
I think the "Other physical type(s) with same unit" column is a little too wide, but I agree! let's get this in and fiddle with some settings when we add x-ref link targets.